danielMunteanu/src/main/java/FilterByType.java

```diff
@@ -0,0 +1,19 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class FilterByType {
```

Comment:
You have create a specific class in order to define a method, it's not bad, however there could be a class related to some kind of utils for exmaples we have Collection and Collections (please find out the differences on internet) . You can create Order and Orders -> in the Orders class you can define methods related to orders utilities.